addons/insignias/CfgUnitInsignia.hpp

```diff
@@ -813,6 +813,18 @@ class CfgUnitInsignia {
         displayName = "Valknut";
         texture = QPATHTOF(data\valknut.paa);
     };
+    class CLASS(Insignia_VTAG) {
+        MACRO_INSIGNIA_COMMON
+        author = "GilleeDoo";
+        displayName = "VTAG";
+        texture = QPATHTOF(data\vtagPatch.paa);
```

Comment:
Actually why `Patch` in the filename?

Reply:
Didn't really think about it, probably should change that.